ui/app/AppLayouts/Profile/ProfileLayout.qml

```diff
@@ -99,15 +100,28 @@ StatusSectionLayout {
         readonly property bool toastClashesWithDirtyBubble: root.Window.width <= 1650 // design
     }
 
+    SettingsEntriesModel {
+        id: settingsEntriesModel
+
+        showWalletEntries: root.store.walletMenuItemEnabled
+        showBackUpSeed: !root.store.privacyStore.mnemonicBackedUp
+
+        syncingBadgeCount: root.store.devicesStore.devicesModel.count
+                           - root.store.devicesStore.devicesModel.pairedCount
```

Comment:
nit: I think I prefer when the operator in on the line before, as it shows clearly that the line is not done